css/mistica-common.css

```diff
@@ -133,3 +249,331 @@
         --mistica-responsive-layout-margin: calc((100vw - 1224px) / 2);
     }
 }
+
+/* Tag */
+[class^='mistica-tag'] {
+    font-weight: var(--mistica-font-weight-indicator);
+    border-radius: var(--mistica-border-radius-indicator);
+    font-size: 0.875rem;
+    line-height: 1.25rem;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    min-width: 56px;
+    padding: 4px 12px;
+    vertical-align: middle;
+    width: fit-content;
+}
+
+.mistica-tag-promo {
+    background: var(--vcolor-tagBackgroundPromo);
+    color: var(--vcolor-tagTextPromo);
+}
+
+.mistica-tag-active {
+    background: var(--vcolor-tagBackgroundActive);
+    color: var(--vcolor-tagTextActive);
+}
+
+.mistica-tag-inactive {
+    background: var(--vcolor-tagBackgroundInactive);
+    color: var(--vcolor-tagTextInactive);
+}
+
+.mistica-tag-success {
+    background: var(--vcolor-tagBackgroundSuccess);
+    color: var(--vcolor-tagTextSuccess);
+}
+
+.mistica-tag-warning {
+    background: var(--vcolor-tagBackgroundWarning);
+    color: var(--vcolor-tagTextWarning);
+}
+
+.mistica-tag-error {
+    background: var(--vcolor-tagBackgroundError);
+    color: var(--vcolor-tagTextError);
+}
+
+/* Buttons */
+[class^='mistica-button'] {
+    display: inline-block;
+    border-radius: var(--mistica-border-radius-button);
+    font-size: var(--mistica-font-size-2);
```

Comment:
font size in small buttons in desktop should not increase.
https://tinyurl.com/254uvufc

Reply:
should be fixed now

Reply:
mmm something is wrong. It is not increasing but the size is not ok

https://tinyurl.com/2apcj4n2

Default button size
mobile: 16px
desktop: 18px

Small button size
mobile: 14px
desktop: 14px

Reply:
I think I fixed it now